M2/Macaulay2/packages/MultigradedBGG.m2

```diff
@@ -403,6 +406,11 @@ doc ///
       toricRR
       toricLL
       stronglyLinearStrand
+   References
+       Text
+       [1] @HREF{"https://arxiv.org/pdf/2202.00402v4","Linear strands of multigraded free resolutions"}@ (with Daniel Erman), Mathematische Annalen 390 (2024), 2707–2725
+       [2] @HREF{"https://arxiv.org/pdf/2108.03345v3","Tate resolutions on toric varieties"}@ (with Daniel Erman), Journal of the European Mathematical Society, published online (2024)
```

Comment:
There's an [`arXiv`](https://macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/SimpleDoc/html/_ar__Xiv.html) function you can use for this if you'd like.